### PR TITLE
BUG Fix incorrect drag-drop handle

### PR DESCRIPTION
**Pull request by tractorcow**
I've also copied across the most current indentation settings from asset-admin.

Screenshot:

image

### CODE REVIEW

**Comment by dhensby** [comment]
So we're using the sort icon as a drag-and-drop icon? Is there a more appropriate icon to use?

**Comment by tractorcow** [comment]
I can't see one without modifying the font file, unless you find one you like.

**Comment by dhensby** [comment]
Where are the font icons even coming from?

**Comment by tractorcow** [comment]
`silverstripe` font that's provided in core.

Open up `/silverstripe-admin/client/src/font/icons-reference.html` in a local checkout to see the list preview. :)

**Comment by sachajudd** [comment]
Currently working on a custom drag & drop icon @tractorcow @dhensby 🙂

**Comment by tractorcow** [comment]
Hi @sachajudd do you think we could use something like https://github.com/primer/octicons instead of raster fonts? Our custom `silverstripe` font is based on a customised subset of these, but I think it wouldn't be too hard to extend this list in custom modules (e.g. silverstripe-extra-icons font).

**Comment by tractorcow** [comment]
FYI we use http://fontastic.me/ to build the font files.

**Comment by sachajudd** [comment]
Hey @tractorcow I'm going to add my icon to the font set that we use for the CMS 🙂

**Comment by tractorcow** [comment]
Nice, let me know if you need any help. :)

**Comment by tractorcow** [comment]
Also can use https://oblador.github.io/react-native-vector-icons/

image

File: templates/Symbiote/GridFieldExtensions/GridFieldOrderableRowsDragHandle.ss [hunk 1]
```diff
@@ -1,2 +1,2 @@
-<span class="handle"><span class="icon"></span></span>
+<div class="handle"><i class="icon font-icon-sort"></i></div>
```
Comment (sachajudd) [inline templates/Symbiote/GridFieldExtensions/GridFieldOrderableRowsDragHandle.ss]: Icon class can be changed to font-icon-drag-handle once https://github.com/silverstripe/silverstripe-admin/pull/219 is merged 🙂

**Comment by dhensby** [comment]
I've made those changes for you @sachajudd

**Pull request merged**